Makefile

```diff
@@ -203,21 +203,21 @@ install-dependencies: $(NODE_TARBALL) $(YARN_TARBALL) $(SERVE_TARBALL)
 .PHONY: install-python-modules
 install-python-modules:
 	#$(_INFO) Installing Python SDK $(_END)
-	cd src/sdk/pynni && python3 setup.py install $(PIP_MODE)
+	cd src/sdk/pynni && $(PIP_INSTALL) $(PIP_MODE) .
```

Comment:
`README.Makefile.md` should be updated because PIP is not optional anymore.

Reply:
At first, I was confused that why 'setup.py install' is used in install target and 'pip install' is used in dev-install target. And it seems nni cannot be uninstalled through pip if it's installed by setup.py, so I change it

Reply:
Because setuptools (setup.py) does not support editable (symlink) mode as far as I know.
Using pip looks fine to me anyway. Just sync the doc.

Reply:
Thanks @liuzhe-lz . @Crysple  could you update the README file accordingly?

Reply:
> Thanks @liuzhe-lz . @Crysple could you update the README file accordingly?

Sure I will